### DISCUSSION TITLE
[@types/node] Which version should I use for Node 16.16.0 + TypeScript 5.0.2?

### DISCUSSION THREAD

**Discussion by pypt** [DefinitelyTyped/DefinitelyTyped#65236] [Issues with a @types package] [answered]
Hello,

I'm a little bit confused by the `@types/node` versioning scheme, and I've searched this discussion board but couldn't find anything.

Could you please tell me, which version of `@types/node` should I use for the Node 16.16.0 + TypeScript 5.0.2 combination?

Thank you in advance!

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment, marked as answer]
Use the last v16.x version.

**Comment by pypt** [comment]
Thank you!
